### PR TITLE
Update game over message

### PR DESCRIPTION
**Pull request by RB1234567890-star**
update

### CODE REVIEW

**Review by shrubsies — COMMENTED** [review]
Looks good.

**Review by RB1234567890-star — COMMENTED** [review]
Looks good.

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (shrubsies) [inline index.html:51]: maybe add something to this line, IDK

Comment (RB1234567890-star) [inline index.html:51]: I'm suing you.

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
```
Comment (shrubsies) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again !</h2>
```

Comment (RB1234567890-star) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again!</h2>
```

**Pull request merged**